package.json

```diff
@@ -56,18 +56,21 @@
     "@folio/stripes-components": "~6.1.0",
     "@folio/stripes-connect": "~5.5.0",
     "@folio/stripes-core": "~4.1.0",
+    "@folio/stripes-final-form": "^2.1.100041",
```

Comment:
Such version exist only on the folioci. 
Please change it to "@folio/stripes-final-form": "^2.1.1",

Reply:
Thanks! I checked that the latest released version is 2.1.0